libs-scala/resources-grpc/src/main/scala/com/daml/resources/grpc/NioEventLoopGroupResourceOwner.scala

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.resources.grpc
+
+import java.util.concurrent.ThreadFactory
+
+import com.daml.resources.{AbstractResourceOwner, HasExecutionContext, Resource}
+import io.netty.channel.nio.NioEventLoopGroup
+
+import scala.concurrent.Future
+import scala.concurrent.duration.FiniteDuration
+
+private[grpc] final class NioEventLoopGroupResourceOwner[Context: HasExecutionContext](
```

Comment:
Please merge this with `com.daml.platform.apiserver.EventLoopGroupOwner`.

Reply:
Addressed in https://github.com/digital-asset/daml/pull/8604/commits/ebf80e3a2974576f1f10e1f2ede2b381b099abfe